server/src/Config.hs

```diff
@@ -59,6 +61,11 @@ configParser environment = Config
                   metavar "HOST:PORT" <>
                   help "If provided, Icepeak collects various metrics and provides them to Prometheus on the given endpoint."
                  ))
+  <*> fmap (fromMaybe 256)
```

Comment:
It's possible to add a default value with the `value` modifier, i.e.

```
(option auto
                       (long "queue-capacity" <>
                        metavar "INTEGER" <>
                        value 256 <>
                        help "Higher values allow more requests to come in within a short amount of time.")
```

looks a bit cleaner then.

Reply:
Ah, nice. :+1: